tensorboard/plugins/text/summary_v2.py

```diff
@@ -26,6 +26,40 @@
 def text(name, data, step=None, description=None):
     """Write a text summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`, `tf.summary.image`.
+
+    Logs text tensor values for later visualization and analysis in TensorBoard.  Like
+    `tf.summary.scalar` points, text points are each associated with a `step` and a
+    `name`.  All the points with the same `name` constitute a time series of text values.
+
+    For Example:
+    ```python
+    test_summary_writer = tf.summary.create_file_writer('test/text_demo_log_dir')
+    with test_summary_writer.as_default():
+        tf.summary.text('first_text', 'hello world!', step=0)
+        tf.summary.text('first_text', 'nice to meet you!', step=1)
+    ```
+
+    The text summary can also contain Markdown, and TensorBoard will render the text
+    as such.
+
+    ```python
+    with test_summary_writer.as_default():
+        table_as_markdown_text = '''
+              | *hello* | *there* |
+              |---------|---------|
+              | this    | is      |
+              | a       | table   |
+        '''
+        tf.summary.text('markdown_text', table_as_markdown_text, step=0)
+    ```
+
+    Since text is tensor valued, each text point may be a tensor of string values.
```

Comment:
Our docs aren't always consistent about writing "tensor" vs "\`Tensor\`", but I see the latter in many of the TF devsite docs.  Should we replace instances in this paragraph with "\`Tensor\`"?

Reply:
done